pkg/controller/servicemesh/memberroll/controller_test.go

```diff
@@ -285,23 +285,19 @@ func TestReconcileReconcilesAddedMember(t *testing.T) {
 			name:                "default",
 			operatorVersion:     operatorVersionDefault,
 			meshVersion:         "",
```

Comment:
this can be removed I guess? the whole field can be removed from the struct

Reply:
No, it's used to initialize `SMCP.spec.version` in line 297

Reply:
Sorry, my bad; only looked at one side of the diff